AWS-Transcription.py

```diff
@@ -1,86 +1,102 @@
-from __future__ import print_function
+#!usr/bin/env python3
+
 import boto3
 import time
-import os
+from distutils.util import strtobool
+from pathlib import Path
 
 #Sets variables
 s3 = boto3.client('s3')
-print('Enter the s3 bucket name')
-s3Bucket = input()
-
-print('Enter name of file in Audio to upload')
-fileName = input()
-filePath = 'Audio/' + fileName
-noExt, fileExt = os.path.splitext(filePath)
-if fileExt == '.mp3':
-    fileExt = 'mp3'
-elif fileExt == '.mp4':
-    fileExt = 'mp4'
-elif fileExt == '.wav':
-    fileExt = 'wav'
-elif fileExt == '.flac':
-    fileExt = 'flac'
-else:
-    print('Invalid format. Convert to mp3 | mp4 | wav | flac')
-    exit()
```

Comment:
This if-elif-else block was rather unnecessary, it was easy to replace with a simple tuple lookup. Also, `exit` shouldn't be used outside REPL sessions, so I replaced it with an exception instead. You could alternatively use `sys.exit`, but that's usually reserved for other things.